core/pxt_blockly_functions.js

```diff
@@ -110,14 +110,14 @@ Blockly.Functions.flyoutCategory = function(workspace) {
  * @private
  */
 Blockly.Functions.addCreateButton_ = function(workspace, xmlList) {
-  var button = Blockly.utils.xml.createElement('button');
+  var button = document.createElement('button');
   var msg = Blockly.Msg.FUNCTION_CREATE_NEW;
   var callbackKey = 'CREATE_FUNCTION';
   var callback = function() {
     Blockly.Functions.createFunctionCallback_(workspace);
   };
   button.setAttribute('text', msg);
-  button.setAttribute('callbackkey', callbackKey);
+  button.setAttribute('callbackKey', callbackKey);
```

Comment:
You’ll have issues with deserializing this in IE. It doesn’t like cases in XML nodes.